docs/software.md

```diff
@@ -59,6 +59,14 @@
 - [**Bitwarden_RS - Unofficial Bitwarden password manager server written in Rust**](../software/cloud/#bitwarden_rs)
 - [**FuguHub - Your Own Personal Cloud Server**](../software/cloud/#fuguhub)
 
+## [Databases & Data Stores](../software/databases/#overview)
+
+- [**MariaDB** - Open Source Relational Database](../software/databases/#mariadb)
+  - [**phpMyAdmin** - SQL Administration Tool for MariaDB](../software/databases/#phpmyadmin)  
```

Comment:
Is this supposed to be indented?

Reply:
I think it was, but to have it really indented, four spaces are required: https://dietpi.com/docs/software/#databases-data-stores
However, I suggest to skip that indentation. Otherwise we'd need to do that for many other software options, like Nextcloud Talk etc. It doesn't make the list look nicer IMO.